### PR TITLE
feat: add dropArgs

### LINKED ISSUE #679: dropArgs
```js
const dropArgs = R.nAry(0);
```

### PR DESCRIPTION
**Pull request by char0n**
Closes #679

### CODE REVIEW

File: src/dropArgs.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { nAry } from 'ramda';
+
+/**
+ * Accepts a function with any arity and returns a function with arity of zero.
+ * The function returned ignores any arguments supplied to it.
```
Comment (srghma) [inline src/dropArgs.js]: The returned function ignores any arguments supplied to it

File: src/dropArgs.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * @category Logic
+ * @sig (...a -> b)-> () -> b
+ * @param {Function} fn The function with any arity
+ * @return {Function} Returns function with arity of zero
+ * @see {@link http://ramdajs.com/docs/#nAry|R.nAry}
+ * @example
+ *
+ * const fn = (a = 1, b = 2) => a + b;
+ *
+ * RA.dropArgs(fn)(3, 4); //=> 3
```
Comment (srghma) [inline src/dropArgs.js]: RA.dropArgs(fn)("ignored1", "ignored2");

File: src/index.d.ts [hunk 3]
```diff
@@ -973,6 +973,12 @@ declare namespace RamdaAdjunct {
         argsPass(combiningPredicate: Function, predicates: Function[]): Function;
         argsPass(combiningPredicate: Function): (predicates: Function[]) => Function;
 
+        /**
+         * Accepts a function with any arity and returns a function with arity of zero.
+         * The function returned ignores any arguments supplied to it.
```
Comment (srghma) [inline src/index.d.ts]: The returned function ignores any arguments supplied to it

File: src/dropArgs.js (hunk 2 above)
Comment (srghma) [reply src/dropArgs.js]: @char0n you missed it here

Comment (char0n) [reply src/dropArgs.js]: Thanks caught it here: https://github.com/char0n/ramda-adjunct/commit/d4c2add6a63787570547a02c16c5e1da0422a228

**Pull request merged**